styles/Footer.module.css

```diff
@@ -0,0 +1,98 @@
+.footer__container {
+  position: relative;
+  padding: 0px 0px 16px;
+  width: 100%;
+  height: 520px;
+}
+.footer__main__container {
+  position: relative;
+  display: flex;
+  width: 100%;
+  height: 375px;
+  align-items: center;
+  border-style: solid none solid;
+  border-width: 1px;
+}
+.footer__main__fylla__container {
+  position: relative;
```

Comment:
I don't see why so many relative positions in this CSS. Maybe there is something I missed?

Reply:
I removed 2 relative fields. But also 2 remain:
1. 
<img width="364" alt="image" src="https://user-images.githubusercontent.com/110534329/219549926-9b0d5119-3b4c-4b99-b78a-c9b03b968efc.png">
The lower left corner has an absolute position, so its father needs a relative position
2. 
<img width="1132" alt="image" src="https://user-images.githubusercontent.com/110534329/219550165-acdd5cef-f5ae-4af0-a54d-12d676802a9e.png">
The copyright and privacy texts have an absolute position, keeping one in the lower left corner, and one in the lower right corner.